server/src/api/modules/name-changes/services/BulkSubmitNameChangesService.ts

```diff
@@ -32,28 +33,24 @@ type BulkSubmitResult = {
 
 async function bulkSubmitNameChanges(payload: BulkSubmitParams): Promise<BulkSubmitResult> {
   const input = inputSchema.parse(payload);
+  const submitted: NameChange[] = [];
```

Comment:
this could actually just be a number count, since we don't seem to anything with the actual array itself, just it's length

could you include this change please? 🙏